src/App.js

```diff
@@ -1,5 +1,5 @@
 import React, { useEffect } from "react"
-import { ConfigProvider, Flex, Layout, message, notification, theme, Typography } from 'antd';
+import { ConfigProvider, Flex, Layout, message, /* notification,  */theme, Typography } from 'antd';
```

Comment:
`notification` seems like cruft - wasn't used anywhere in the app.  left it commented out, but it probably can be straight nuked.

Reply:
Right, that's gone now